doc/ipython-notebooks/gaussian_process/variational_classifier.ipynb

```diff
@@ -629,14 +733,380 @@
+      "    n_boundary=50\n",
+      "    x1_boundary = np.linspace(x_train[0,:].min()-1, x_train[0,:].max()+1, n_boundary)\n",
+      "    x2_boundary = np.linspace(x_train[1,:].min()-1, x_train[1,:].max()+1, n_boundary)\n",
+      "    x_boundary = np.asarray(list(product(x1_boundary, x2_boundary))).T\n",
+      "\n",
+      "    features_boundary=RealFeatures(x_boundary)\n",
+      "    \n",
+      "    for idx,kernel_log_sigma in enumerate(kernel_log_sigmas):\n",
+      "        #train a GPC model given traning data points\n",
+      "        (gpc, name)=train(inference, linesearch, likelihood, x_train, y_train, kernel_log_sigma, kernel_log_scale)\n",
+      "        #obtain the probabilities of being positive label(s) given new data point(s)\n",
+      "        prbs=gpc.get_probabilities(features_boundary)\n",
+      "\n",
+      "        for choice in [0,1]:\n",
+      "             #decision boundary\n",
+      "             plots[idx][choice].contour(x1_boundary, x2_boundary, np.reshape(prbs, (n_boundary, n_boundary)).T, levels=[0.5], colors=('blue'))  \n",
+      "             #training points or testing points with true positive tag\n",
+      "             plots[idx][choice].scatter(x1s[choice][1],x2s[choice][1], c='red', alpha=0.5)\n",
+      "             #training points or testing points with true negative tag\n",
+      "             plots[idx][choice].scatter(x1s[choice][0],x2s[choice][0], c='yellow', alpha=0.5)\n",
+      "             plots[idx][choice].set_xlabel(\"At1\")\n",
+      "             plots[idx][choice].set_ylabel(\"At2\")\n",
+      "             plots[idx][choice].axis(\"equal\")\n",
+      "             type_name=\"training\"\n",
+      "             if choice==1:\n",
+      "                type_name=\"testing\"\n",
+      "             if idx==0:\n",
+      "                fit_condition=\"over-fit\"\n",
+      "             elif idx==1:\n",
+      "                fit_condition=\"right-fit\"\n",
+      "             else:\n",
+      "                fit_condition=\"under-fit\"                           \n",
+      "             plots[idx][choice].set_title(\"Decision boundary (blue) \\n of %s on %s points (%s)\"%(name, type_name,fit_condition))\n",
+      "\n",
+      "    plt.show()"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "inf='../../../data/toy/banana.dat'\n",
+      "plot(inf)"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "heading",
+     "level": 2,
+     "metadata": {},
+     "source": [
+      "<a id=\"largescale\">Inference Methods for sparse GPC Models</a> <a href='#toc'>[TOP]</a>"
+     ]
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "TO DO: Theory of large-scale inference methods for sparse Gaussian processes will be discussed in a new notebook.\n",
```

Comment:
An intro here would be nice. Say that you are reproducing the same experiment with large scale GPs ... not just theory, also intuition and high level roadmap